kubemarine/procedures/check_paas.py

```diff
@@ -1163,12 +1176,40 @@ def calico_version(criteria: str) -> str:
             tc.success(results='valid')
 
 
+def calico_apiserver_health_status(cluster: KubernetesCluster) -> None:
+    """
+    This test verifies the Calico API server health.
+
+    :param cluster: KubernetesCluster object
+    :return: None
+    """
+    with TestCase(cluster, '225', "Calico", "API server health status") as tc:
```

Comment:
I would prefer to not break original check number sequence and to make this check as latest +1. Since the purpose of each check number is to be constant across all versions to avoid confusion.

Reply:
In this case checks for Calico will have numbers far from each other. Also it is not clear how to reflect this in documentation, whether the checks should be described one after another with jumping numbers, or they should be sorted according to the numbers and be described in different parts of the document. I prefer localizing of related information, so we will have jumping numbers in the documentation. This will change the current approach, as the numbers are sorted now. https://github.com/Netcracker/KubeMarine/blob/feature/calico_apiserver/documentation/Kubecheck.md
What will you prefer?

Reply:
Done in https://github.com/Netcracker/KubeMarine/pull/510/commits/b13115d43364e59a08902b67df735c1f16ba917a